ProcessLib/InitialCondition.h

```diff
@@ -45,8 +45,8 @@ class UniformInitialCondition : public InitialCondition
 	UniformInitialCondition(double const value) : _value(value)
 	{
 	}
-
-	virtual double getValue(MeshLib::Node const&) const override
+	virtual double getValue(MeshLib::Node const&,
+	                        int const /* tuple_size */) const override
```

Comment:
Have a look on the freshly pushed code....
There will be more race conditions otherwise...